dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title>New Dataset - GDV</template>
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading GDV data...</p>
+        <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+      </div>
```

Comment:
Add button is now deactivated. However switching between Yes and No is lagging, so that for a time both a active. Aslo the error message should be modified to be more descriptive of the issue.
![image](https://github.com/d-fine/Dataland/assets/104447806/0af7b10f-3e94-402d-b976-b7bb99601fc7)

Reply:
Lag seems to come from formkit and the browser having to load all the components if yes is toggled. No real solution came to our minds, so closing this for now